osu.Game.Rulesets.Osu.Tests/TestSceneOsuTouchInput.cs

```diff
@@ -150,6 +150,34 @@ public void TestSimpleInput()
             assertKeyCounter(1, 1);
         }
 
+        [Test]
+        public void TestPositionalTrackingAfterLongDistanceTravelled()
+        {
+            // When a single touch has already travelled enough distance on screen, it should remain as the positional
+            // tracking touch until released (unless a direct touch occurs).
+
+            beginTouch(TouchSource.Touch1);
+
+            assertKeyCounter(1, 0);
+            checkPressed(OsuAction.LeftButton);
+            checkPosition(TouchSource.Touch1);
+
+            // cover some distance
+            beginTouch(TouchSource.Touch1, new Vector2(0));
+            beginTouch(TouchSource.Touch1, new Vector2(9999));
+            beginTouch(TouchSource.Touch1, new Vector2(0));
+            beginTouch(TouchSource.Touch1, new Vector2(9999));
+            beginTouch(TouchSource.Touch1);
+
+            beginTouch(TouchSource.Touch2);
```

Comment:
I'd like to see a test where `Touch2` moves around. To show that even if `Touch2` moves >100 px, `Touch1` is still tracked (if that is the intended behaviour).